### PR TITLE
Update start up tests

### PR DESCRIPTION
**Pull request by troydai**
1. Update to rc1-update1
2. Trim down samples to two: BasicKestrel and StarterMvc
3. Update utilities

Note:
1. The sample is created from rc1 template and works with rc1-update1. The MVC doesn't work under rc2 latest (something thrown from Razor, the sample probably need to be updated.) So it sticked to rc1-update1 for now.
2. Results are written to local json file. When Taylor has integrate the reporting mechanism the test cases logging can be streamlined.
``` JSON
{
  "Info": {
    "Measurer": "WebApplicationFirstRequest",
    "CommandFilename": "C:\\Users\\Xueyuan\\.dnx\\runtimes\\dnx-clr-win-x86.1.0.0-rc1-update1\\bin\\dnx.exe",
    "CommandArguments": "kestrel",
    "Url": "http://localhost:5001/",
    "Timeout": "00:01:00",
    "Successful rate": "1",
    "Successful iteration": "10"
  },
  "Data": {
    "Mean": "2969.5",
    "SD": "1409.74696311076"
  }
}
```

### CODE REVIEW

**Comment by troydai** [comment]
@NTaylorMullen @Eilon

**Comment by troydai** [comment]
/cc @muratg 

Since it is not produce code, I’ll merge this soon.

File: NuGet.Config [hunk 1]
```diff
@@ -1,7 +1,7 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
-    <add key="AspNetVNext" value="https://www.myget.org/F/aspnetvnext/" />
-    <add key="NuGet" value="https://nuget.org/api/v2/" />
+    <add key="AspNetVNext" value="https://www.myget.org/F/aspnetcirelease/api/v3/index.json" />
```
Comment (NTaylorMullen) [inline NuGet.Config]: Why not aspnetcidev? Also, should lowercase the `.config` for this file.

**Comment by NTaylorMullen** [comment]
Unless @muratg or @Eilon feel differently. I'd just merge it in. Hacking for now, can clean up later.

File: NuGet.Config (hunk 1 above)
Comment (muratg) [inline NuGet.Config]: Yeah, should use the dev feed in my opinion.

**Comment by muratg** [comment]
:shipit:

**Pull request merged**